homu/utils.py

```diff
@@ -46,3 +52,85 @@ def remove_url_keys_from_json(json):
 def lazy_debug(logger, f):
     if logger.isEnabledFor(logging.DEBUG):
         logger.debug(f())
+
+def maybe_call(dkt, key, func):
+    if key in dkt:
+        return func(dkt[key])
+
+# this thing could be a macros in the better world
+def ignore(func, error=Exception, logger=None):
+    try:
+        return func()
+    except error as e:
+        if logger:
+            logger.warning(e)
+
+def update_in(dkt, key, func):
+    if key in dkt:
+        dkt[key] = func(dkt[key])
+    return dkt
+
+def merge_dicts(fst, snd=None, **kwargs):
```

Comment:
Alternative solution to chain several stuff `merge_dicts(dict(asd=1), dict(qwe=2), asd=1, qwe=3)` might save some lines 
```
from itertools import chain

def merge_dicts(*args, **kwargs):
    merged = chain(
        chain(*(d.items() for d in args)),
        kwargs.items()
    )

    return dict(merged)
```